### PR TITLE
Add link to new client library

### PR DESCRIPTION
**Pull request by jesse-c**
Point to the new official client library. After this, we can archive this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**NB: We've released an official [client library in Ruby](https://github.com/duffelhq/duffel-api-ruby) for the Duffel API. We recommend using that over this quickstart example.**
```
Comment (nlopes) [inline README.md:1]:
```suggestion
**NB: We've released an official [client library in Ruby](https://github.com/duffelhq/duffel-api-ruby) for the Duffel API. We recommend using that repository [examples](https://github.com/duffelhq/duffel-api-ruby/tree/main/examples) over this quickstart example.**
```

**Review by nlopes — APPROVED** [review]

**Review by sgerrand — APPROVED** [review]
LGTM. 👍

File: README.md (hunk 1 above)
Comment (sgerrand) [reply README.md:1]:
```suggestion
**NB: We've released an official [client library in Ruby](https://github.com/duffelhq/duffel-api-ruby) for the Duffel API. We recommend using the [examples](https://github.com/duffelhq/duffel-api-ruby/tree/main/examples) in that repository over this quickstart example.**
```

Comment (jesse-c) [reply README.md:1]: I'm going to quickly edit this with "that repository's".

**Pull request merged**